src/Models/Page.php

```diff
@@ -3,6 +3,7 @@
 namespace TCG\Voyager\Models;
 
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Auth;
```

Comment:
I believe we can remove this PR. I had created this on Master, issued the PR. Then I created a separate branch for a different PR and it had this change on it. So it's already up.